### ISSUE TITLE
Docker takes 10+minutes to start

### ISSUE THREAD

**Issue by raoulh** [alexjustesen/speedtest-tracker#500]
**Describe the bug**
I run docker on a basic debian bullseye machine. All standard default options.
When I start the docker-compose up, the docker starts but the web UI is not available before at least 10-15min.

This is my docker-compose file (from doc)
```
version: '3.3'
services:
    speedtest-tracker:
        container_name: speedtest-tracker
        ports:
            - '8080:80'
            - '8443:443'
        environment:
            - PUID=1000
            - PGID=1000
        volumes:
            - './config:/config'
        image: 'ghcr.io/alexjustesen/speedtest-tracker:latest'
        restart: unless-stopped
```

**Expected behavior**
Direct start without waiting so long

**Environment (please complete the following information):**
 - OS: Debian bullseye
-  Architecture: amd64
 - Version: ghcr.io/alexjustesen/speedtest-tracker:latest

**Logs**
I added thos env:
```
DEBUG_OUTPUT=true
S6_VERBOSITY=5
```

And here is the log from the start:
```
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier cron of type longrun
speedtest-tracker  | s6-rc-compile: tracing: from cron: adding identifier laravel-automations of type unknown
speedtest-tracker  | s6-rc-compile: info: parsing /etc/s6-overlay/s6-rc.d/debug-nginx
speedtest-tracker  | s6-rc-compile: info: debug-nginx has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier debug-nginx of type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from debug-nginx: adding identifier runas-user of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier runas-user was already declared with type unknown
speedtest-tracker  | s6-rc-compile: tracing: dependency from debug-nginx to runas-user (48)
speedtest-tracker  | s6-rc-compile: tracing: dependency from debug-nginx to s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: info: parsing /etc/s6-overlay/s6-rc.d/laravel-automations
speedtest-tracker  | s6-rc-compile: info: laravel-automations has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier laravel-automations of type oneshot
speedtest-tracker  | s6-rc-compile: tracing: previously encountered identifier laravel-automations now has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from laravel-automations: adding identifier runas-user of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier runas-user was already declared with type unknown
speedtest-tracker  | s6-rc-compile: tracing: from laravel-automations: adding identifier configure-ssl of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier configure-ssl was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: tracing: dependency from laravel-automations to runas-user (48)
speedtest-tracker  | s6-rc-compile: tracing: dependency from laravel-automations to configure-ssl (34)
speedtest-tracker  | s6-rc-compile: tracing: dependency from laravel-automations to s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: info: parsing /etc/s6-overlay/s6-rc.d/msmtp
speedtest-tracker  | s6-rc-compile: info: msmtp has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier msmtp of type oneshot
speedtest-tracker  | s6-rc-compile: tracing: dependency from msmtp to s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: info: parsing /etc/s6-overlay/s6-rc.d/nginx
speedtest-tracker  | s6-rc-compile: info: nginx has type longrun
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier nginx of type longrun
speedtest-tracker  | s6-rc-compile: tracing: from nginx: adding identifier php-fpm of type unknown
speedtest-tracker  | s6-rc-compile: tracing: from nginx: adding identifier configure-ssl of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier configure-ssl was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from nginx: adding identifier debug-nginx of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier debug-nginx was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: info: parsing /etc/s6-overlay/s6-rc.d/php-fpm
speedtest-tracker  | s6-rc-compile: info: php-fpm has type longrun
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier php-fpm of type longrun
speedtest-tracker  | s6-rc-compile: tracing: previously encountered identifier php-fpm now has type longrun
speedtest-tracker  | s6-rc-compile: tracing: from php-fpm: adding identifier runas-user of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier runas-user was already declared with type unknown
speedtest-tracker  | s6-rc-compile: tracing: from php-fpm: adding identifier laravel-automations of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier laravel-automations was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: info: parsing /etc/s6-overlay/s6-rc.d/queue-worker
speedtest-tracker  | s6-rc-compile: info: queue-worker has type longrun
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier queue-worker of type longrun
speedtest-tracker  | s6-rc-compile: tracing: from queue-worker: adding identifier laravel-automations of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier laravel-automations was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: info: parsing /etc/s6-overlay/s6-rc.d/runas-user
speedtest-tracker  | s6-rc-compile: info: runas-user has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier runas-user of type oneshot
speedtest-tracker  | s6-rc-compile: tracing: previously encountered identifier runas-user now has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: dependency from runas-user to s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: info: parsing /etc/s6-overlay/s6-rc.d/user
speedtest-tracker  | s6-rc-compile: info: user has type bundle
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier user of type bundle
speedtest-tracker  | s6-rc-compile: tracing: from user: adding identifier queue-worker of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier queue-worker was already declared with type longrun
speedtest-tracker  | s6-rc-compile: tracing: from user: adding identifier cron of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier cron was already declared with type longrun
speedtest-tracker  | s6-rc-compile: tracing: from user: adding identifier msmtp of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier msmtp was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from user: adding identifier debug-nginx of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier debug-nginx was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from user: adding identifier configure-ssl of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier configure-ssl was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from user: adding identifier nginx of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier nginx was already declared with type longrun
speedtest-tracker  | s6-rc-compile: tracing: from user: adding identifier php-fpm of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier php-fpm was already declared with type longrun
speedtest-tracker  | s6-rc-compile: tracing: from user: adding identifier laravel-automations of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier laravel-automations was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from user: adding identifier runas-user of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier runas-user was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: info: parsing /etc/s6-overlay/s6-rc.d/user2
speedtest-tracker  | s6-rc-compile: info: user2 has type bundle
speedtest-tracker  | s6-rc-compile: tracing: from /etc/s6-overlay/s6-rc.d: adding identifier user2 of type bundle
speedtest-tracker  | s6-rc-compile: info: parsing /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources/base
speedtest-tracker  | s6-rc-compile: info: base has type bundle
speedtest-tracker  | s6-rc-compile: tracing: from /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources: adding identifier base of type bundle
speedtest-tracker  | s6-rc-compile: tracing: from base: adding identifier fix-attrs of type unknown
speedtest-tracker  | s6-rc-compile: tracing: from base: adding identifier legacy-cont-init of type unknown
speedtest-tracker  | s6-rc-compile: info: parsing /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources/fix-attrs
speedtest-tracker  | s6-rc-compile: info: fix-attrs has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources: adding identifier fix-attrs of type oneshot
speedtest-tracker  | s6-rc-compile: tracing: previously encountered identifier fix-attrs now has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: dependency from fix-attrs to s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: info: parsing /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources/legacy-cont-init
speedtest-tracker  | s6-rc-compile: info: legacy-cont-init has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources: adding identifier legacy-cont-init of type oneshot
speedtest-tracker  | s6-rc-compile: tracing: previously encountered identifier legacy-cont-init now has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from legacy-cont-init: adding identifier fix-attrs of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier fix-attrs was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: tracing: dependency from legacy-cont-init to fix-attrs (145)
speedtest-tracker  | s6-rc-compile: tracing: dependency from legacy-cont-init to s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: info: parsing /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources/legacy-services
speedtest-tracker  | s6-rc-compile: info: legacy-services has type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources: adding identifier legacy-services of type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from legacy-services: adding identifier user of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier user was already declared with type bundle
speedtest-tracker  | s6-rc-compile: tracing: from legacy-services: adding identifier base of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier base was already declared with type bundle
speedtest-tracker  | s6-rc-compile: tracing: dependency from legacy-services to user (129)
speedtest-tracker  | s6-rc-compile: tracing: dependency from legacy-services to base (140)
speedtest-tracker  | s6-rc-compile: tracing: dependency from legacy-services to s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: info: parsing /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources/top
speedtest-tracker  | s6-rc-compile: info: top has type bundle
speedtest-tracker  | s6-rc-compile: tracing: from /package/admin/s6-overlay-3.1.2.1/etc/s6-rc/sources: adding identifier top of type bundle
speedtest-tracker  | s6-rc-compile: tracing: from top: adding identifier user of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier user was already declared with type bundle
speedtest-tracker  | s6-rc-compile: tracing: from top: adding identifier legacy-services of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier legacy-services was already declared with type oneshot
speedtest-tracker  | s6-rc-compile: tracing: from top: adding identifier base of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier base was already declared with type bundle
speedtest-tracker  | s6-rc-compile: tracing: from top: adding identifier user2 of type unknown
speedtest-tracker  | s6-rc-compile: tracing: identifier user2 was already declared with type bundle
speedtest-tracker  | s6-rc-compile: info: making bundles for pipelines
speedtest-tracker  | s6-rc-compile: info: resolving bundle names
speedtest-tracker  | s6-rc-compile: info: converting bundle array
speedtest-tracker  | s6-rc-compile: info: resolving service names
speedtest-tracker  | s6-rc-compile: tracing: cron depends on oneshot laravel-automations (8)
speedtest-tracker  | s6-rc-compile: tracing: nginx depends on longrun php-fpm (4)
speedtest-tracker  | s6-rc-compile: tracing: nginx depends on oneshot configure-ssl (6)
speedtest-tracker  | s6-rc-compile: tracing: nginx depends on oneshot debug-nginx (7)
speedtest-tracker  | s6-rc-compile: tracing: php-fpm depends on oneshot runas-user (10)
speedtest-tracker  | s6-rc-compile: tracing: php-fpm depends on oneshot laravel-automations (8)
speedtest-tracker  | s6-rc-compile: tracing: queue-worker depends on oneshot laravel-automations (8)
speedtest-tracker  | s6-rc-compile: tracing: configure-ssl depends on oneshot runas-user (10)
speedtest-tracker  | s6-rc-compile: tracing: configure-ssl depends on longrun s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: tracing: debug-nginx depends on oneshot runas-user (10)
speedtest-tracker  | s6-rc-compile: tracing: debug-nginx depends on longrun s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: tracing: laravel-automations depends on oneshot runas-user (10)
speedtest-tracker  | s6-rc-compile: tracing: laravel-automations depends on oneshot configure-ssl (6)
speedtest-tracker  | s6-rc-compile: tracing: laravel-automations depends on longrun s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: tracing: msmtp depends on longrun s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: tracing: runas-user depends on longrun s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: tracing: fix-attrs depends on longrun s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: tracing: legacy-cont-init depends on oneshot fix-attrs (11)
speedtest-tracker  | s6-rc-compile: tracing: legacy-cont-init depends on longrun s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: tracing: legacy-services depends on bundle user
speedtest-tracker  | s6-rc-compile: tracing: legacy-services depends on bundle base
speedtest-tracker  | s6-rc-compile: tracing: legacy-services depends on longrun s6rc-oneshot-runner (0)
speedtest-tracker  | s6-rc-compile: info: converting service dependency array
speedtest-tracker  | s6-rc-compile: info: checking database correctness
speedtest-tracker  | s6-rc-compile: info: writing compiled information to /run/s6/db
speedtest-tracker  | s6-rc-compile: info: writing /run/s6/db/n
speedtest-tracker  | s6-rc-compile: info: writing /run/s6/db/resolve.cdb
speedtest-tracker  | s6-rc-compile: info: writing /run/s6/db/db
speedtest-tracker  | s6-rc-compile: info: writing /run/s6/db/servicedirs
speedtest-tracker  | s6-rc: info: bringing selected services up
speedtest-tracker  | s6-rc: info: service s6rc-oneshot-runner: starting
speedtest-tracker  | s6-rc: info: service s6rc-oneshot-runner successfully started
speedtest-tracker  | s6-rc: info: service fix-attrs: starting
speedtest-tracker  | s6-ipcclient: connected to /run/s6-rc/servicedirs/s6rc-oneshot-runner/s
speedtest-tracker  | s6-rc: info: service runas-user: starting
speedtest-tracker  | s6-ipcclient: connected to /run/s6-rc/servicedirs/s6rc-oneshot-runner/s
speedtest-tracker  | s6-rc: info: service msmtp: starting
speedtest-tracker  | s6-ipcclient: connected to /run/s6-rc/servicedirs/s6rc-oneshot-runner/s
speedtest-tracker  | s6-rc: info: service fix-attrs successfully started
speedtest-tracker  | s6-rc: info: service legacy-cont-init: starting
speedtest-tracker  | s6-ipcclient: connected to /run/s6-rc/servicedirs/s6rc-oneshot-runner/s
speedtest-tracker  | s6-rc: info: service msmtp successfully started
speedtest-tracker  | s6-rc: info: service legacy-cont-init successfully started


### Here it does not do anything until 10-15 minutes, then it finally starts the web app:


speedtest-tracker  |
speedtest-tracker  | --------------------------------------------------------------------
speedtest-tracker  |
speedtest-tracker  |  _______                       __ __                __
speedtest-tracker  | |     __|.-----.-----.-----.--|  |  |_.-----.-----.|  |_
speedtest-tracker  | |__     ||  _  |  -__|  -__|  _  |   _|  -__|__ --||   _|
speedtest-tracker  | |_______||   __|_____|_____|_____|____|_____|_____||____|
speedtest-tracker  |          |__|
speedtest-tracker  |  _______                   __
speedtest-tracker  | |_     _|.----.---.-.----.|  |--.-----.----.
speedtest-tracker  |   |   |  |   _|  _  |  __||    <|  -__|   _|
speedtest-tracker  |   |___|  |__| |___._|____||__|__|_____|__|
speedtest-tracker  |
speedtest-tracker  |
speedtest-tracker  | --------------------------------------------------------------------
speedtest-tracker  |
speedtest-tracker  | To support Speedtest Tracker visit:
speedtest-tracker  | https://github.com/alexjustesen/speedtest-tracker
speedtest-tracker  | -------------------------------------
speedtest-tracker  | GID/UID
speedtest-tracker  | -------------------------------------
speedtest-tracker  |
speedtest-tracker  | User uid:    2000
speedtest-tracker  | User gid:    2000
speedtest-tracker  | -------------------------------------
speedtest-tracker  |
speedtest-tracker  | s6-rc: info: service runas-user successfully started
speedtest-tracker  | s6-rc: info: service debug-nginx: starting
speedtest-tracker  | s6-ipcclient: connected to /run/s6-rc/servicedirs/s6rc-oneshot-runner/s
speedtest-tracker  | s6-rc: info: service configure-ssl: starting
speedtest-tracker  | s6-ipcclient: connected to /run/s6-rc/servicedirs/s6rc-oneshot-runner/s
speedtest-tracker  | 🔒 SSL_MODE has been set to MIXED, setting the web server to work in HTTP + HTTPS...
speedtest-tracker  | 🏃‍♂️ An SSL key was not detected, so I'll generate a self-signed SSL certificate pair for you...
speedtest-tracker  | 🔥 NGINX and PHP-FPM are in DEBUG MODE 🔥
speedtest-tracker  | s6-rc: info: service debug-nginx successfully started
speedtest-tracker  | ........+..+...+......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*...+...+......+.....+.+........+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.........+.+...+......+.....+....+............+...+..+....+.....+...+......+..........+.....+.+...+..+.............+...+.....+.......+....................+.+..+....+...+..........................+..........+.................+...+...................+...+..+.+..+............+......+.+...+...+...+......+...+....................+.+...........+.+............+...+.....+..........+..+............+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
speedtest-tracker  | .+..+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.........+.....+......+...+.+......+...+.....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*..........+............+......+..+.........+.............+...........+......+.+..+.+..............+.+.....+...............+.+...........+............+.......+........+.............+.........+......+...+......+........+.+.....+......+.............+........+...+.......+..............+.............+...........+.........+.+......+........+...+...+.......+.....+.......+...........+.......+.....+.............+.........+...........+....+..+..........+.....+....+..+....+...+..+..........+...........+...+......+......+.........+......+....+...+.....+......+....+...+..+............+.+.........+...+.....+......+.......+............+......+...........+....+............+...+.........+...........+.+..............+.+...+..+.....................+.........+.+...+..+....+...+..+....+......+........+...+...+....+...+..+.+......+......+..+.+..+..........+............+..+.+...+..+.........+.............+.....+......+.......+.........+........+......+......+.............+...+.....+.......+...+.....+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
speedtest-tracker  | -----
speedtest-tracker  | s6-rc: info: service configure-ssl successfully started
speedtest-tracker  | s6-rc: info: service laravel-automations: starting
speedtest-tracker  | s6-ipcclient: connected to /run/s6-rc/servicedirs/s6rc-oneshot-runner/s
speedtest-tracker  |
speedtest-tracker  | 🐇  Configuring Speedtest Tracker...
speedtest-tracker  |
speedtest-tracker  | ✅  Database already exists.
speedtest-tracker  |
speedtest-tracker  | 🔗  Creating database symlink...
speedtest-tracker  | ✅  Database symlink created.
speedtest-tracker  |
speedtest-tracker  | ✅  Environment file exists.
speedtest-tracker  |
speedtest-tracker  | 🔗  Creating symlinks for config and log files...
speedtest-tracker  | ✅  Symlinks created.
speedtest-tracker  |
speedtest-tracker  | ✅  App key exists
speedtest-tracker  |
speedtest-tracker  | 🔒  Fixing app path file permissions...
speedtest-tracker  | ✅  Permissions fixed.
speedtest-tracker  |
speedtest-tracker  | 🔐  Linking the storage...
speedtest-tracker  | ✅  Storage linked.
speedtest-tracker  |
speedtest-tracker  | 💰  Building the cache...
speedtest-tracker  | ✅  Cache set.
speedtest-tracker  |
speedtest-tracker  | 🚛  Migrating the database...
speedtest-tracker  | ✅  Database migrated.
speedtest-tracker  |
speedtest-tracker  | ✅  All set, Speedtest Tracker started.
speedtest-tracker  |
speedtest-tracker  | s6-rc: info: service laravel-automations successfully started
speedtest-tracker  | s6-rc: info: service queue-worker: starting
speedtest-tracker  | s6-rc: info: service php-fpm: starting
speedtest-tracker  | s6-rc: info: service cron: starting
speedtest-tracker  | s6-rc: info: service cron successfully started
speedtest-tracker  | s6-rc: info: service queue-worker successfully started
speedtest-tracker  | [19-Mar-2023 14:16:52.436399] DEBUG: pid 131, fpm_stdio_save_original_stderr(), line 81: saving original STDERR fd: dup()
speedtest-tracker  | [19-Mar-2023 14:16:52.436511] DEBUG: pid 131, fpm_log_open(), line 48: open access log (/proc/self/fd/1)
speedtest-tracker  | [19-Mar-2023 14:16:52.436846] DEBUG: pid 131, fpm_scoreboard_init_main(), line 38: got clock tick '100'
speedtest-tracker  | [19-Mar-2023 14:16:52.436942] DEBUG: pid 131, fpm_signals_init_main(), line 221: Unblocking all signals
speedtest-tracker  | [19-Mar-2023 14:16:52.437047] DEBUG: pid 131, fpm_socket_af_inet_socket_by_addr(), line 306: Found address for ::, socket opened on ::
speedtest-tracker  | [19-Mar-2023 14:16:52.437083] DEBUG: pid 131, fpm_event_init_main(), line 359: event module is epoll and 41 fds have been reserved
speedtest-tracker  | [19-Mar-2023 14:16:52.437202] NOTICE: pid 131, fpm_init(), line 83: fpm is running, pid 131
speedtest-tracker  | [19-Mar-2023 14:16:52.437231] DEBUG: pid 131, fpm_event_loop(), line 387: 23488 bytes have been reserved in SHM
speedtest-tracker  | [19-Mar-2023 14:16:52.437237] NOTICE: pid 131, fpm_event_loop(), line 388: ready to handle connections
speedtest-tracker  | [19-Mar-2023 14:16:52.438818] DEBUG: pid 131, fpm_systemd_heartbeat(), line 67: have notify start to systemd
speedtest-tracker  | [19-Mar-2023 14:16:52.438842] NOTICE: pid 131, fpm_systemd_heartbeat(), line 74: systemd monitor interval set to 10000ms
speedtest-tracker  | [19-Mar-2023 14:16:53.291927] DEBUG: pid 131, fpm_children_make(), line 405: blocking signals before child birth
speedtest-tracker  | [19-Mar-2023 14:16:53.294057] DEBUG: pid 131, fpm_children_make(), line 429: unblocking signals, child born
speedtest-tracker  | [19-Mar-2023 14:16:53.294194] DEBUG: pid 131, fpm_children_make(), line 435: [pool www] child 157 started
speedtest-tracker  | [19-Mar-2023 14:16:53.294217] DEBUG: pid 131, fpm_pctl_on_socket_accept(), line 553: [pool www] got accept without idle child available .... I forked
speedtest-tracker  | [19-Mar-2023 14:16:53.294229] DEBUG: pid 131, fpm_event_loop(), line 440: event module triggered 1 events
speedtest-tracker  | [19-Mar-2023 14:16:53.297512] DEBUG: pid 131, fpm_event_loop(), line 440: event module triggered 1 events
speedtest-tracker  | s6-rc: info: service php-fpm successfully started
speedtest-tracker  | s6-rc: info: service nginx: starting
speedtest-tracker  | ❌ There seems to be a failure in checking the web server + PHP-FPM. Here's the response:
speedtest-tracker  |
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 171#171: epoll add event: fd:7 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 171#171: epoll add event: fd:8 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 171#171: epoll add event: fd:9 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 171#171: epoll add event: fd:10 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 172#172: epoll add event: fd:7 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 172#172: epoll add event: fd:8 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 172#172: epoll add event: fd:9 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 172#172: epoll add event: fd:10 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 175#175: epoll add event: fd:7 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 175#175: epoll add event: fd:8 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 175#175: epoll add event: fd:9 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 175#175: epoll add event: fd:10 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 173#173: epoll add event: fd:7 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 174#174: epoll add event: fd:7 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 173#173: epoll add event: fd:8 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 174#174: epoll add event: fd:8 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 173#173: epoll add event: fd:9 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 174#174: epoll add event: fd:9 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 173#173: epoll add event: fd:10 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 174#174: epoll add event: fd:10 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 176#176: epoll add event: fd:7 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 176#176: epoll add event: fd:8 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 176#176: epoll add event: fd:9 op:1 ev:10000001
speedtest-tracker  | 2023/03/19 14:16:53 [debug] 176#176: epoll add event: fd:10 op:1 ev:10000001
speedtest-tracker  | [19-Mar-2023 14:16:53.437832] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: accept on 0.0.0.0:80, ready: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: posix_memalign: 000055F2AD671150:512 @16
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 accept: 127.0.0.1:57158 fd:18
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer add: 18: 60000:10349838
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 reusable connection: 1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 epoll add event: fd:18 op:1 ev:80002001
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http wait request handler
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 malloc: 000055F2AD651990:1024
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 recv: eof:0, avail:-1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 recv: fd:18 77 of 1024
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 reusable connection: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 posix_memalign: 000055F2AD7498D0:4096 @16
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http process request line
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http request line: "GET /ping HTTP/1.1"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http uri: "/ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http args: ""
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http exten: ""
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 posix_memalign: 000055F2AD6613F0:4096 @16
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http process request header line
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http header: "Host: localhost"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http header: "User-Agent: curl/7.81.0"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http header: "Accept: */*"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http header done
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer del: 18: 10349838
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 generic phase: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 rewrite phase: 1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 test location: "/"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 test location: "ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 test location: ~ "\.php$"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 test location: ~ "\.(?:css(\.map)?|js(\.map)?|jpe?g|png|gif|ico|cur|heic|webp|tiff?|mp3|m4a|aac|ogg|midi?|wav|mp4|mov|webm|mpe?g|avi|ogv|flv|wmv)$"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 test location: ~ "\.(?:svgz?|ttf|ttc|otf|eot|woff2?)$"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 test location: ~ "/\.(?!well-known)"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 using configuration "/ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http cl:-1 max:2147483648
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 rewrite phase: 3
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 post rewrite phase: 4
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 generic phase: 5
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 generic phase: 6
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 generic phase: 7
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 access phase: 8
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 access phase: 9
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 access phase: 10
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 post access phase: 11
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 generic phase: 12
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 generic phase: 13
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http init upstream, client timer: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 epoll add event: fd:18 op:3 ev:80002005
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "QUERY_STRING"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "QUERY_STRING: "
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "REQUEST_METHOD"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "GET"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "REQUEST_METHOD: GET"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "CONTENT_TYPE"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "CONTENT_TYPE: "
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "CONTENT_LENGTH"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "CONTENT_LENGTH: "
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "SCRIPT_NAME"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "/ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "SCRIPT_NAME: /ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "REQUEST_URI"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "/ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "REQUEST_URI: /ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "DOCUMENT_URI"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "/ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "DOCUMENT_URI: /ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "DOCUMENT_ROOT"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "/var/www/html/public"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "DOCUMENT_ROOT: /var/www/html/public"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "SERVER_PROTOCOL"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "HTTP/1.1"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "SERVER_PROTOCOL: HTTP/1.1"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "REQUEST_SCHEME"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "http"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "REQUEST_SCHEME: http"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: ""
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "GATEWAY_INTERFACE"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "CGI/1.1"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "GATEWAY_INTERFACE: CGI/1.1"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "SERVER_SOFTWARE"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "nginx/"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "1.18.0"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "SERVER_SOFTWARE: nginx/1.18.0"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "REMOTE_ADDR"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "127.0.0.1"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "REMOTE_ADDR: 127.0.0.1"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "REMOTE_PORT"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "57158"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "REMOTE_PORT: 57158"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "REMOTE_USER"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "REMOTE_USER: "
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "SERVER_ADDR"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "127.0.0.1"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "SERVER_ADDR: 127.0.0.1"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "SERVER_PORT"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "80"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "SERVER_PORT: 80"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "SERVER_NAME"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script var: "_"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "SERVER_NAME: _"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "REDIRECT_STATUS"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "200"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "REDIRECT_STATUS: 200"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "SCRIPT_NAME"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "/ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "SCRIPT_NAME: /ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "SCRIPT_FILENAME"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http script copy: "/ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "SCRIPT_FILENAME: /ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "HTTP_HOST: localhost"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "HTTP_USER_AGENT: curl/7.81.0"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 fastcgi param: "HTTP_ACCEPT: */*"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http cleanup add: 000055F2AD74A8B0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 get rr peer, try: 1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 stream socket 19
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 epoll add connection: fd:19 ev:80002005
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 connect to 127.0.0.1:9000, fd:19 #2
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream connect: -2
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 posix_memalign: 000055F2AD737970:128 @16
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer add: 19: 60000:10349838
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http finalize request: -4, "/ping?" a:1, c:2
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http request count:2 blk:0
speedtest-tracker  | [19-Mar-2023 14:16:54.373824] DEBUG: pid 131, fpm_event_loop(), line 440: event module triggered 1 events
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http run request: "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream check client, write event:1, "/ping"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream request: "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream send request handler
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream send request
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream send request body
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 chain writer buf fl:0 s:528
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 chain writer in: 000055F2AD661FF8
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 writev: 528 of 528
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 chain writer out: 0000000000000000
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer del: 19: 10349838
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer add: 19: 5000:10294842
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream request: "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream process header
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 malloc: 000055F2AD656BB0:4096
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 recv: eof:0, avail:-1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 recv: fd:19 176 of 4096
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 01
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 06
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 00
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 01
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 00
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 95
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 03
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 00
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record length: 149
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi parser: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi header: "Content-type: text/plain;charset=UTF-8"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi parser: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi header: "Expires: Thu, 01 Jan 1970 00:00:00 GMT"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi parser: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi header: "Cache-Control: no-cache, no-store, must-revalidate, max-age=0"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi parser: 1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi header done
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 posix_memalign: 000055F2AD657BC0:4096 @16
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 HTTP/1.1 200 OK
speedtest-tracker  | Server: nginx
speedtest-tracker  | Date: Sun, 19 Mar 2023 14:16:54 GMT
speedtest-tracker  | Content-Type: text/plain;charset=UTF-8
speedtest-tracker  | Transfer-Encoding: chunked
speedtest-tracker  | Connection: keep-alive
speedtest-tracker  | Vary: Accept-Encoding
speedtest-tracker  | Expires: Thu, 01 Jan 1970 00:00:00 GMT
speedtest-tracker  | Cache-Control: no-cache, no-store, must-revalidate, max-age=0
speedtest-tracker  | X-Frame-Options: SAMEORIGIN
speedtest-tracker  | X-Content-Type-Options: nosniff
speedtest-tracker  | Referrer-Policy: no-referrer-when-downgrade
speedtest-tracker  | Strict-Transport-Security: max-age=31536000; includeSubDomains
speedtest-tracker  |
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write new buf t:1 f:0 000055F2AD657BE0, pos 000055F2AD657BE0, size: 460 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http write filter: l:0 f:0 s:460
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http cacheable: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream process upstream
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe read upstream: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe preread: 23
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe buf free s:0 t:1 f:0 000055F2AD656BB0, pos 000055F2AD656C49, size: 23 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe length: -1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe write downstream: 1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe write busy: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe write: out:0000000000000000, f:0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe read upstream: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe buf free s:0 t:1 f:0 000055F2AD656BB0, pos 000055F2AD656C49, size: 23 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe length: -1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer: 19, old: 10294842, new: 10294842
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream request: "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream dummy handler
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream request: "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream process upstream
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe read upstream: 1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 readv: eof:1, avail:-1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 readv: 1, last:3920
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe recv chain: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe buf free s:0 t:1 f:0 000055F2AD656BB0, pos 000055F2AD656C49, size: 23 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe length: -1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 input buf #0 000055F2AD656C49
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 01
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 03
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 00
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 01
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 00
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 08
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 00
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record byte: 00
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi record length: 8
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http fastcgi sent end request
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 input buf 000055F2AD656C49 4
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe write downstream: 1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe write downstream flush in
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http output filter "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http copy filter: "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http postpone filter "/ping?" 000055F2AD662348
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http chunk: 4
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write old buf t:1 f:0 000055F2AD657BE0, pos 000055F2AD657BE0, size: 460 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write new buf t:1 f:0 000055F2AD6623C8, pos 000055F2AD6623C8, size: 3 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write new buf t:1 f:0 000055F2AD656BB0, pos 000055F2AD656C49, size: 4 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write new buf t:0 f:0 0000000000000000, pos 000055F2AD0722E8, size: 2 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http write filter: l:0 f:0 s:469
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http copy filter: 0 "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 pipe write downstream done
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer: 19, old: 10294842, new: 10294842
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream exit: 0000000000000000
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 finalize http upstream request: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 finalize http fastcgi request
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 free rr peer 1 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 close http upstream connection: 19
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 free: 000055F2AD737970, unused: 48
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer del: 19: 10294842
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 reusable connection: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http upstream temp fd: -1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http output filter "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http copy filter: "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http postpone filter "/ping?" 00007FFDFCD82360
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http chunk: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write old buf t:1 f:0 000055F2AD657BE0, pos 000055F2AD657BE0, size: 460 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write old buf t:1 f:0 000055F2AD6623C8, pos 000055F2AD6623C8, size: 3 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write old buf t:1 f:0 000055F2AD656BB0, pos 000055F2AD656C49, size: 4 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write old buf t:0 f:0 0000000000000000, pos 000055F2AD0722E8, size: 2 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 write new buf t:0 f:0 0000000000000000, pos 000055F2AD0722E5, size: 5 file: 0, size: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http write filter: l:1 f:0 s:474
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http write filter limit 0
speedtest-tracker  | [19-Mar-2023 14:16:54.375821] DEBUG: pid 131, fpm_event_loop(), line 440: event module triggered 1 events
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 writev: 474 of 474
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http write filter 0000000000000000
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http copy filter: 0 "/ping?"
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http finalize request: 0, "/ping?" a:1, c:1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 set http keepalive handler
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http close request
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http log handler
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 free: 000055F2AD656BB0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 free: 000055F2AD7498D0, unused: 8
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 free: 000055F2AD6613F0, unused: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 free: 000055F2AD657BC0, unused: 2784
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 free: 000055F2AD651990
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 hc free: 0000000000000000
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 hc busy: 0000000000000000 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 tcp_nodelay
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 reusable connection: 1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer add: 18: 65000:10354842
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 http keepalive handler
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 malloc: 000055F2AD651990:1024
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 recv: eof:1, avail:-1
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 recv: fd:18 0 of 1024
speedtest-tracker  | 2023/03/19 14:16:54 [info] 171#171: *1 client 127.0.0.1 closed keepalive connection
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 close http connection: 18
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 event timer del: 18: 10354842
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 reusable connection: 0
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 free: 000055F2AD651990
speedtest-tracker  | 2023/03/19 14:16:54 [debug] 171#171: *1 free: 000055F2AD671150, unused: 120
speedtest-tracker  | s6-rc: info: service nginx successfully started
speedtest-tracker  | s6-rc: info: service legacy-services: starting
speedtest-tracker  | s6-ipcclient: connected to /run/s6-rc/servicedirs/s6rc-oneshot-runner/s
speedtest-tracker  | s6-rc: info: service legacy-services successfully started
speedtest-tracker  | [19-Mar-2023 14:16:54.437984] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:16:55.439205] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:16:56.440467] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:16:57.441689] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:16:58.442901] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:16:59.444137] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:17:00.445440] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:17:01.446682] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:17:02.447085] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:17:03.448298] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:17:04.448877] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:17:05.450106] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 1 spare children
speedtest-tracker  | [19-Mar-2023 14:17:05.470788] DEBUG: pid 131, fpm_event_loop(), line 440: event module triggered 2 events
speedtest-tracker  | [19-Mar-2023 14:17:05.470888] DEBUG: pid 131, fpm_got_signal(), line 82: received SIGCHLD
speedtest-tracker  | [19-Mar-2023 14:17:05.470913] DEBUG: pid 131, fpm_event_loop(), line 440: event module triggered 1 events
speedtest-tracker  | [19-Mar-2023 14:17:05.470958] DEBUG: pid 131, fpm_children_bury(), line 260: [pool www] child 157 has been killed by the process management after 12.176828 seconds from start
speedtest-tracker  | [19-Mar-2023 14:17:06.452120] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:07.453304] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:08.454527] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:09.455672] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:10.456879] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:11.458100] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:12.458512] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:13.459739] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:14.460940] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:15.462200] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:16.463429] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:17.464665] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:18.466037] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:19.467323] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:20.468593] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:21.469825] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:22.470246] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:23.471505] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:24.472824] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:25.474048] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:26.475268] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:27.476503] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:28.477732] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:29.479009] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:30.480294] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:31.481584] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:32.481992] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:33.483217] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:34.484493] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:35.485814] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:36.487005] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:37.488254] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:38.489495] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:39.490746] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:40.492008] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:41.493344] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:42.493824] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:43.495065] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:44.496288] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children
speedtest-tracker  | [19-Mar-2023 14:17:45.497568] DEBUG: pid 131, fpm_pctl_perform_idle_server_maintenance(), line 382: [pool www] currently 0 active children, 0 spare children

```

**Comment by alexjustesen** [comment]
What's the specs on your server and/or is it on a VM running low memory or CPU (vCPU) limited?

**Comment by raoulh** [comment]
Running in a VM with plenty of CPU core and RAM, this is not the issue at all

**Comment by qudiqudi** [comment]
I experience the same. Just that docker logs is not outputting anything for a while (10 mins?) The container is up and "healthy", but webpage not reachable.
My system: 30 container stack on 4 core 64G RAM debian.

**Comment by smbonn2005** [comment]
Experienced the same issue on a reboot on a K3S cluster. Pod up and healthy, nothing in logs and webserver doesn't come up until 11 minutes after pod is up. Container is pinned to latest image.

**Comment by mendoncart** [comment]
Experienced the same issue. Taking a lot of time to load up.

**Comment by alexjustesen** [comment]
FPM is known for running slow atm, I'll be targeting this issue once I get back from healing. #537

**Comment by alexjustesen** [comment]
Latest versions of the base image seem to be using less resources. I'll continue to monitor as I make more fundamental changes but closing this for the time being
